src/Compilers/CSharp/Test/Semantic/Semantics/OperatorTests.cs

```diff
@@ -8650,6 +8642,56 @@ void Test(dynamic x)
             compilation = compilation.WithOptions(TestOptions.ReleaseDll.WithOverflowChecks(true));
             semanticModel = compilation.GetSemanticModel(tree);
 
+            var symbols2 = (from node2 in nodes select (IMethodSymbol)semanticModel.GetSymbolInfo(node2).Symbol).ToArray();
+            foreach (var symbol2 in symbols2)
+            {
+                Assert.False(symbol2.IsCheckedBuiltin);
+                Assert.True(((ITypeSymbol)symbol2.ContainingSymbol).IsDynamic());
+                Assert.Null(symbol2.ContainingType);
+            }
+
+            for (int i = 0; i < symbols1.Length; i++)
+            {
+                Assert.Equal(symbols1[i], symbols2[i]);
+            }
+        }
+
+        [Fact]
+        public void DynamicBinaryIntrinsicSymbols2()
+        {
+            var source =
+@"
+class Module1
+{
+    void Test(dynamic x)
+    {
+        var z1 = x + 0;
+        var z2 = 0 + x;
+    }
+}";
+
+            var compilation = CreateCompilation(source, options: TestOptions.ReleaseDll.WithOverflowChecks(false));
+
+            var tree = compilation.SyntaxTrees.Single();
+            var semanticModel = compilation.GetSemanticModel(tree);
+
+            var nodes = (from node in tree.GetRoot().DescendantNodes()
+                         select node as BinaryExpressionSyntax).
+                         Where(node => node is not null).ToArray();
```

Comment:
Nit: `tree.GetRoot.DescendentNodes().OfType<BinaryExpressionSyntax>().ToArray()`

Reply:
Ah, I see it was just a copy/paste. Never mind, this is fine.